packages/bls12-381-unix/bls12-381-unix.2.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: """\
+UNIX version of BLS12-381 primitives implementing the virtual package
+bls12-381 with blst backend"""
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.8.4"}
+  "dune-configurator" {build}
+  "zarith" {>= "1.10" & < "2.0"}
+  "bls12-381" {= version}
+  "hex"
+  "alcotest" {with-test}
+  "integers"
+  "bisect_ppx" {with-test & >= "2.5"}
+  "ff-pbt" {>= "0.6.0" & < "0.7.0" & with-test}
+]
+available: arch != "ppc64"
```

Comment:
```suggestion
available: arch != "ppc64" & arch != "arm32" & arch != "x86_32"
```
Due to
```
=== ERROR while compiling bls12-381-unix.2.0.1 ===============================#
# context              2.1.2 | linux/arm32 | ocaml-base-compiler.4.13.1 | pinned(https://gitlab.com/dannywillems/ocaml-bls12-381/-/archive/2.0.1/ocaml-bls12-381-2.0.1.tar.bz2)
# path                 ~/.opam/4.13/.opam-switch/build/bls12-381-unix.2.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -j 159 -p bls12-381-unix @install
# exit-code            1
# env-file             ~/.opam/log/bls12-381-unix-2714-8305a5.env
# output-file          ~/.opam/log/bls12-381-unix-2714-8305a5.out
### output ###
#   build_blst src/blst/dllblst.so,src/blst/libblst.a
# + cc -O -fno-builtin -fPIC -Wall -Wextra -Werror -Wno-missing-braces -c ./src/server.c
# + cc -O -fno-builtin -fPIC -Wall -Wextra -Werror -Wno-missing-braces -c ./build/assembly.S
# + ar rc libblst.a assembly.o server.o
# + cc -shared -o libblst.so -Wl,--whole-archive,libblst.a,--no-whole-archive -O -fno-builtin -fPIC -Wall -Wextra -Werror -Wno-missing-braces -Wl,-Bsymbolic,--version-script=/opam-tmp/build9de696.dune/ld.blst.2772
#           cp src/blst/dllblst.so,src/blst/libblst.a
# /bin/cp: cannot stat 'libblst/libblst.dylib': No such file or directory
#          gcc src/blst/blst_bindings_stubs.o (exit 1)
# (cd _build/default/src/blst && /usr/bin/gcc -O2 -fno-strict-aliasing -fwrapv -pthread -fPIC -D_FILE_OFFSET_BITS=64 -Wno-incompatible-pointer-types -g -I /home/opam/.opam/4.13/lib/ocaml -I /home/opam/.opam/4.13/lib/bigarray-compat -I /home/opam/.opam/4.13/lib/bls12-381 -I /home/opam/.opam/4.13/lib/cstruct -I /home/opam/.opam/4.13/lib/ff-sig -I /home/opam/.opam/4.13/lib/hex -I /home/opam/.opam/4.13/lib/integers -I /home/opam/.opam/4.13/lib/stdlib-shims -I /home/opam/.opam/4.13/lib/zarith -o blst_bindings_stubs.o -c blst_bindings_stubs.c)
# blst_bindings_stubs.c:21:2: error: #error "No suitable OCaml type available for representing size_t values"
#    21 | #error "No suitable OCaml type available for representing size_t values"
#       |  ^~~~~
# blst_bindings_stubs.c: In function 'caml_blst_p1_mult_stubs':
# blst_bindings_stubs.c:594:16: warning: implicit declaration of function 'ctypes_size_t_val' [-Wimplicit-function-declaration]
#   594 |                ctypes_size_t_val(size));
#       |                ^~~~~~~~~~~~~~~~~
```

Reply:
Thanks. I forgot to add this commit for 2.0.1, but it is on master upstream.